4-appfactory/modules/app-group-baseline/main.tf

```diff
@@ -37,10 +37,30 @@ locals {
     }
   )
 
-  org_ids = distinct([for env in var.envs : env.org_id])
+  org_ids           = distinct([for env in var.envs : env.org_id])
+  use_csr           = var.cloudbuildv2_repository_config.repo_type == "CSR"
+  service_repo_name = var.cloudbuildv2_repository_config.repositories[var.service_name].repository_name
 }
 
 
+module "cloudbuild_repositories" {
+  count = local.use_csr ? 0 : 1
+  # TODO: update git URL to registry format after release is available
+  source = "git::https://github.com/terraform-google-modules/terraform-google-bootstrap.git//modules/cloudbuild_repo_connection?ref=a6072e0db92d976f1535dab767ad6b4331bcb4ef"
```

Comment:
Can we update with registry release now? https://github.com/terraform-google-modules/terraform-google-bootstrap/releases/tag/v10.0.0